test/parallel/test-event-emitter-onMultiple.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const common = require('../common');
+
+// This test ensures that it is possible to add a listener to multiple events at once
+
+const assert = require('assert');
+const EventEmitter = require('../../lib/events');
+const myEE = new EventEmitter();
+
+async function main() {
+
+    // Input
+    const inputArray = ["a", "b", Symbol('symbol')];
+    const emptyFunction = function() {};
+
+    // Test
+    await myEE.onMultiple(inputArray, emptyFunction);
```

Comment:
Why is this awaited? I'm also not sure this actually tests the feature itself.

Reply:
> Why is this awaited?

Good point. I'll fix it. Thanks.

>  I'm also not sure this actually tests the feature itself.

To me, it seems like you're not conviced that using `.eventNames(`) isn't the proper way to test the code. I mean, I could agree on some degree. How do you propose to change it?